intellij/src/nl/tudelft/watchdog/intellij/logic/InitializationManager.java

```diff
@@ -32,7 +32,13 @@
     private final Persister toTransferPersister;
     private final Persister statisticsPersister;
 
+    /**
```

Comment:
Don't the comments suggest that a more appropriate naming of these three should be done? Also, what's the difference between the WatchDogEventManager and the intervalManager now?

Reply:
I think this is mostly an artifact of the usage of `*Manager`. In general, "manager" is very generic and hard to grasp. The distinction between watchdog events, intervals and debug events is ambiguous as well. Rather than resolve that in this PR, I would like to discuss with you better names and architecture to resolve this ambiguation. WDYT?

Reply:
I agree on the manager part, but this will be hard to change.

What about the following?

WatchDogEvent -> IDEEvent
TrackingEvent -> WatchDogEvent
Interval -> WatchDogInterval

If you still want to do this as part of a separate PR (or have different ideas on the rename), that is more than fine with me.

Reply:
Merging because this is now part of #275. Thx, Tim!